### PR TITLE
Kotlin gen on windows

### PR DESCRIPTION
**Pull request by kkocel**

### CODE REVIEW

File: buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy [hunk 1]
```diff
@@ -29,37 +29,39 @@ import java.nio.file.Path
  * Task that generates Kotlin bindings for the Rx*.java implementations
  *
  * To debug this, you can use the following command and attach a remote debugger to port 5005
- * ./gradlew clean build --no-daemon -Dorg.gradle.debug=true
- */
+ * ./gradlew clean build --no-daemon -Dorg.gradle.debug=true*/
```
Comment (JakeWharton) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: revert

File: buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy [hunk 2]
```diff
@@ -98,15 +100,13 @@ class KotlinGenTask extends SourceTask {
...
-
     }, kClass)
 
     kClass.generate(outputDir)
   }
 
   /**
-   * Represents a kotlin file that corresponds to a Java file/class in an RxBinding module
-   */
+   * Represents a kotlin file that corresponds to a Java file/class in an RxBinding module*/
```
Comment (JakeWharton) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: Revert

File: buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy [hunk 3]
```diff
@@ -144,8 +144,7 @@ class KotlinGenTask extends SourceTask {
   }
 
   /**
-   * Represents a method implementation that needs to be wired up in Kotlin
-   */
+   * Represents a method implementation that needs to be wired up in Kotlin*/
```
Comment (JakeWharton) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: Revert

File: buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy [hunk 4]
```diff
@@ -168,30 +167,48 @@ class KotlinGenTask extends SourceTask {
...
+          }
 
-          // Remove any trailing whitespace
-          .replaceAll(/(?m)\s+$/, "")
-          .trim()
+      // Remove any trailing whitespace
+          .
+          replaceAll(/(?m)\s+$/, "")
+          .
+          trim()
```
Comment (JakeWharton) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: Revert all this formatting

File: buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy [hunk 5]
```diff
@@ -29,37 +29,39 @@ import java.nio.file.Path
...
-      "android.support.annotation.NonNull"
-  )
+      "android.support.annotation.NonNull")
 
   @TaskAction
   def generate(IncrementalTaskInputs inputs) {
     // Clear things out first to make sure no stragglers are left
-    File outputDir = new File("${project.projectDir}-kotlin/src/main/kotlin")
+    File outputDir = new File(
+        "${project.projectDir}-kotlin" + SLASH + "src" + SLASH + "main" + SLASH + "kotlin")
```
Comment (JakeWharton) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: You can use Groovy's string interpolation for this
```groovy
"${project.projectDir}-kotlin${SLASH}src${SLASH}main${SLASH}kotlin"
```

File: buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy [hunk 6]
```diff
@@ -29,37 +29,39 @@ import java.nio.file.Path
...
    * {@link #resolveKotlinTypeByName}
    */
-  private static final List<String> IGNORED_IMPORTS = ImmutableList.of(
-      "java.util.List",
+  private static final List<String> IGNORED_IMPORTS = ImmutableList.of("java.util.List",
       "android.annotation.TargetApi",
       "android.support.annotation.CheckResult",
-      "android.support.annotation.NonNull"
-  )
+      "android.support.annotation.NonNull")
```
Comment (JakeWharton) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: Revert formatting here

File: buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy [hunk 7]
```diff
@@ -168,30 +168,48 @@ class KotlinGenTask extends SourceTask {
...
 
-          // Remove any trailing whitespace
-          .replaceAll(/(?m)\s+$/, "")
-          .trim()
+      // Remove any trailing whitespace
+          .
+          replaceAll(/(?m)\s+$/, "")
+          .
+          trim()
     }
```
Comment (kkocel) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: ok, do you have any formatter for groovy?

Comment (JakeWharton) [inline buildSrc/src/main/groovy/com/jakewharton/rxbinding/project/KotlinGenTask.groovy]: No. I'll port this to Kotlin eventually.

**Comment by JakeWharton** [comment]
Thanks for fixing this.

**Pull request merged**